develop/buf-breaking.sh

```diff
@@ -25,6 +25,9 @@ set -eu -o pipefail
 : "${INTERNAL_BINPB:=proto/image.bin}"
 : "${MAIN_BRANCH:=main}"
 
+: "${COMMIT:=$(git rev-parse HEAD)}"
+: "${PR_BASE_COMMIT:=$(git merge-base HEAD main)}"
```

Comment:
In CI the main branch won't be present so this would fail. I guess in CI it'll set PR_BASE_COMMIT, so that's ok and intended?

Reply:
Yes, this is mostly so it can pass if run locally. Otherwise, it doesn't work.